src/ui/hexedit.cc

```diff
@@ -1341,24 +1346,49 @@ void HexEdit::saveChunkToFile(const QString& path) {
   saveDataToFile(byte_offset, size, path);
 }
 
+/**
+ * The data is saved to temporary file and then moved to destined location,
```

Comment:
`a temporary file`

Reply:
`to destined location` -> `to the destination`

Reply:
Done